File: testdata/eval/java/Sample.java
package sample;

import java.util.List;

public class UserService {
    public String getUser(int id) {
        return "user-" + id;
    }
}

class RequestHandler {
    public void handleRequest() {
        UserService svc = new UserService();
        svc.getUser(1);
    }
}

File: testdata/eval/c/sample.c
#include <stdio.h>
#include <string.h>

typedef struct {
    int id;
} UserService;

void get_user(UserService *svc, char *buf) {
    sprintf(buf, "user-%d", svc->id);
}

void handle_request() {
    UserService svc = {1};
    char buf[64];
    get_user(&svc, buf);
}
